docs/insomnia/custom-themes.md

```diff
@@ -85,16 +93,27 @@ module.exports = {
 The following `style` options are available. Each of these can hold their own `background`, `foreground`, and `highlight` options.
 
 * appHeader
-* paneHeader
-* editor
 * dialog
-* transparentOverlay
-* sidebar
+* dialogFooter
+* dialogHeader
+* dropdown
+* editor
 * link
+* overlay
+* pane
+* paneHeader
+* sidebar
+* sidebarHeader
+* sidebarList
+* tooltip
+* transparentOverlay
 
 ## Custom CSS
 
-In addition to baseline edits and sub-components, you can add custom CSS.
+In addition to baseline edits and sub-components, you can add custom CSS using the `rawCss` option.
+
+{:.alert .alert-primary}
+**Note**: Generally, opt to use predefined parameters to customize your theme rather than using custom CSS. This ensures that your theme will not break in the future as we make changes to Insomnia.
```

Comment:
```suggestion
**Note**: Generally, opt to use predefined properties to customize your theme rather than using custom CSS. This ensures that your theme will not break in the future as we make changes to Insomnia.
```